summertime/model/single_doc/multilingual/base_multilingual_model.py

```diff
@@ -0,0 +1,58 @@
+from summertime.model.single_doc.base_single_doc_model import SingleDocSummModel
+
+import urllib.request
+import fasttext
+
+
+class MultilingualSummModel(SingleDocSummModel):
+
+    lang_tag_dict = None
```

Comment:
What does this do? Maybe it's better to add a single-line comment and also possibly type annotations?

For example, it would be better to understand if you have things like `lang_tag_dict: Dict[str, str]` (if you intend it to be an "str->str" dict) so your teammates know what to put in this dict.